src/shared/sam/debugger/pythonSamDebug.ts

```diff
@@ -219,6 +219,14 @@ export async function invokePythonLambda(
         WAIT_FOR_DEBUGGER_MESSAGES.PYTHON,
         WAIT_FOR_DEBUGGER_MESSAGES.PYTHON_IKPDB,
     ])
+
+    // Make debugpy output log information if our loglevel is at 'debug'
+    // This is done in 'invokePythonLambda' instead of 'makePythonDebugConfig'
+    // since tests will fail if our loglevel is at 'debug'
+    if (!config.noDebug && !config.useIkpdb && getLogger().logLevelEnabled('debug')) {
+        config.debugArgs![0] += ' --debug'
+    }
```

Comment:
why not do this in `makePythonDebugConfig()`. Yeah it will require updating tests, but... the tests always run with "debug" log-level so it's an easy edit.

Reply:
Oh if that's the case then yeah it's an easy fix.

Reply:
I'll add a test to verify that there is no 'debug' option when log level is not debug too.